src/test/java/com/commercetools/sync/types/utils/typeactionutils/BuildLocalizedEnumUpdateActionsTest.java

```diff
@@ -0,0 +1,195 @@
+        final List<UpdateAction<Type>> updateActions = buildLocalizedEnumValuesUpdateActions(
+            FIELD_NAME_1,
+            ENUM_VALUES_ABC,
+            ENUM_VALUES_CAB
+        );
+
+        assertThat(updateActions).containsExactly(
+            ChangeLocalizedEnumValueOrder.of(FIELD_NAME_1, asList(
+                ENUM_VALUE_C.getKey(),
+                ENUM_VALUE_A.getKey(),
+                ENUM_VALUE_B.getKey()
+            ))
+        );
+    }
+
+    @Test
+    public void buildLocalizedEnumUpdateActions_WithRemovedAndDifferentOrder_ShouldBuildChangeOrderActions() {
+        final List<UpdateAction<Type>> updateActions = buildLocalizedEnumValuesUpdateActions(
+            FIELD_NAME_1,
+            ENUM_VALUES_ABC,
+            ENUM_VALUES_CB
+        );
+
+        // remove enum value actions not exists for type resources
+        assertThat(updateActions).containsExactly(
+            ChangeLocalizedEnumValueOrder.of(FIELD_NAME_1, asList(
+                ENUM_VALUE_C.getKey(),
+                ENUM_VALUE_B.getKey()
+            ))
+        );
+    }
+
+    @Test
+    public void buildLocalizedEnumUpdateActions_WithAddedAndDifferentOrder_ShouldBuildChangeOrderAndAddActions() {
+        final List<UpdateAction<Type>> updateActions = buildLocalizedEnumValuesUpdateActions(
+            FIELD_NAME_1,
+            ENUM_VALUES_ABC,
+            ENUM_VALUES_ACBD
+        );
+
+        assertThat(updateActions).containsExactly(
+            AddLocalizedEnumValue.of(FIELD_NAME_1, ENUM_VALUE_D),
+            ChangeLocalizedEnumValueOrder.of(FIELD_NAME_1, asList(
+                ENUM_VALUE_A.getKey(),
+                ENUM_VALUE_C.getKey(),
+                ENUM_VALUE_B.getKey(),
+                ENUM_VALUE_D.getKey()
+            ))
+        );
+    }
+
+    @Test
+    public void buildLocalizedEnumUpdateActions_WithAddedEnumValueInBetween_ShouldBuildChangeOrderAndAddActions() {
+        final List<UpdateAction<Type>> updateActions = buildLocalizedEnumValuesUpdateActions(
+            FIELD_NAME_1,
+            ENUM_VALUES_ABC,
+            ENUM_VALUES_ADBC
+        );
+
+        assertThat(updateActions).containsExactly(
+            AddLocalizedEnumValue.of(FIELD_NAME_1, ENUM_VALUE_D),
+            ChangeLocalizedEnumValueOrder.of(FIELD_NAME_1, asList(
+                ENUM_VALUE_A.getKey(),
+                ENUM_VALUE_D.getKey(),
+                ENUM_VALUE_B.getKey(),
+                ENUM_VALUE_C.getKey()
+            ))
+        );
+    }
+
+    @Test
+    public void buildLocalizedEnumUpdateActions_WithAddedRemovedAndDifOrder_ShouldBuildAddAndOrderEnumValueActions() {
+        final List<UpdateAction<Type>> updateActions = buildLocalizedEnumValuesUpdateActions(
+            FIELD_NAME_1,
+            ENUM_VALUES_ABC,
+            ENUM_VALUES_CBD
+        );
+
+        // remove enum value actions not exists for type resources
```

Comment:
not exists -> does not exist

Reply:
Is there an issue opened for that? Would be nice to have a github issue for it todo later, that is also next to the TODO comment here so it can be easier to find and fix later.

Also would be nice to link the support ticket in CTP with the github issue.

Reply:
We discussed this, Currently, we have 4 issues, the status of these are waiting in CTP backlog and [1 new issue](
https://github.com/commercetools/commercetools-sync-java/issues/339) on our repository. Also, I've updated the code java docs (just public methods) so it will be easy to find and fix in the future. You can check from here: https://github.com/commercetools/commercetools-sync-java/pull/333/commits/cb0f24e7369ab16eb8916f2d29a8f55b4949b2d7